### DISCUSSION TITLE
way to opt out of "The scheduled workflow is disabled because there hasn't been activity in this repository in the last 60 days"

### DISCUSSION THREAD

**Discussion by marklemay** [community/community#86087] [Actions]
### Select Topic Area

Product Feedback

### Body

Similar issue to that described in https://stackoverflow.com/questions/67184368/prevent-scheduled-github-actions-from-becoming-disabled .
Part of the reason I pay for the pro-accounts is to have a few old projects rebuilt regularly.  It would be nice if there was a way to keep some workflows active, without having to resort to 3rd party hacks or messing with fake commits.

**Comment by jidanni** [comment]
How about at least a way to opt-out of the reminder email about it?

OK, I think I found it! I should go to my https://github.com/jidanni/gdal/settings/actions and select "Disable Actions" and maybe that will put an end to those boring emails!

**Reply by marklemay** [reply]
the issue is not the emails. the issue is that the cron jobs are not run.
